csharp-design-patterns/AggregateDesign/Test/OrderAggregateUnitTest.cs

```diff
@@ -0,0 +1,196 @@
+﻿using AggregateDesign.Domain;
+
+namespace Test
+{
+    public class OrderAggregateUnitTest
+    {
+        [Fact]
+        public void WhenOrderContainsItems_ThenOrderTotalReturnsPrice()
+        {
+            Order order = new();
+            order.AddItem("Cloudsoft Women's Running Shoes", 2, 59.99M);
+
+            Assert.Equal(119.98M, order.OrderTotal);
+        }
+
+        [Fact]
+        public void WhenAddingOrderPayment_ThenPaidAmountincreases()
```

Comment:
AmountIncreases instead of Amountincreases (upper camel case)

Reply:
fixed